apps/ewallet_api/lib/ewallet_api/v1/controllers/transaction_request_controller.ex

```diff
@@ -32,6 +35,30 @@ defmodule EWalletAPI.V1.TransactionRequestController do
     |> respond(conn)
   end
 
+  @spec cancel_for_user(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def cancel_for_user(conn, %{"formatted_id" => formatted_id}) do
+    with {:ok, request} <- TransactionRequestFetcher.get(formatted_id) || {:error, :unauthorized},
```

Comment:
```suggestion
    with {:ok, request} <- TransactionRequestFetcher.get(formatted_id),
```

Remove unreachable condition